src/data/MetaDataItem.py

```diff
@@ -20,6 +20,7 @@ def __init__(self, **kwargs):
         self.enum_tags = kwargs.get("enum_tags", [])
         self.is_cancelled = kwargs.get("is_cancelled", False)
         self.is_split_url = kwargs.get("is_split_url", False)
+        self.to_be_deleted = kwargs.get("to_be_deleted", False)
```

Comment:
Looks like this field will not go to the database, nor get set if the MetaDataItem is copied.
Is this intentional?

Reply:
Good catch, I'll fix this